tileview/src/main/java/com/qozix/tileview/tiles/Tile.java

```diff
@@ -254,8 +262,13 @@ void reset() {
     }
     mState = State.UNASSIGNED;
     mRenderTimeStamp = null;
-    if( mBitmap != null && !mBitmap.isRecycled() ) {
-      mBitmap.recycle();
+    if( mBitmap != null ) {
+      BitmapRecycler recycler = mBitmapRecyclerReference.get();
```

Comment:
Since the one `#execute` calls the other. that field is set to a `WeakReference<BitmapRecycler>`, whether or not `recycler` is null:

```
mBitmapRecyclerReference = new WeakReference<>( runnable );
```